Description: verbose error message when the serie file does not exist
 patchfns.in(cat_series): Displays an error message when the $SERIES
 file does not exist, and when run  in verbose mode.
Bug-Debian: http://bugs.debian.org/369908 
Upstream-status: submitted 2014-01-18

---
 quilt/scripts/patchfns.in |    3 +++
 1 file changed, 3 insertions(+)

Index: b/quilt/scripts/patchfns.in
===================================================================
--- a/quilt/scripts/patchfns.in
+++ b/quilt/scripts/patchfns.in
@@ -344,6 +344,9 @@
 		sed -e '/^#/d' -e 's/^[ '$'\t'']*//' \
 		    -e 's/[ '$'\t''].*//' -e '/^$/d' $SERIES
 	else
+		if [ "$opt_verbose" ]; then
+			echo $"No series file found\n" >&2
+		fi
 		return 1
 	fi
 }
